system/operations/pool_get_logs.yml

```
@@ -27,25 +27,30 @@
       src: /var/lib/indy/sandbox/node*.json
       dest: /tmp/logs/node_info
       mode: pull
+    ignore_errors: yes
 
   - name: Fetch node logs
     synchronize:
       src: /var/log/indy/sandbox/
```

```suggestion
      src: /var/log/indy/
```
With this change `ignore_errors` won't be needed at least for this task